### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by mickychetta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,73 @@
...
+
+<img src="https://spinningup.openai.com/en/latest/_images/math/99621d5bcaccd056d6ca3aeb48a27bf8cc0e640c.svg" alt="loss function" width="650">
+
+<br>
+
+# Observations
+Our observations are given to us within the `ObservationsFromGrid` tag in the `get_mission_xml()` function and the `get_observation()` function. Our observations are given to us in the form of a multi dimensional numpy array. Our observation grid represents the 2 x 5 x 5 area surrounding the agent. The observation space that we feed to the trainer is made to differentiate between the different types of ores. We use these numerical values to represent the different ores in the observation space:
+
+```
+self.blocks_dict = {
```
Comment (lilphunky) [inline README.md:24]: Include lava

File: README.md [hunk 2]
```diff
@@ -1,8 +1,73 @@
...
+<i>Any other block is represented as a 0.</i>
+
+<br>
+
+![ores](docs/images/ores.jpg)
+
+<br>
+
+# Rewards
+Our agent will be rewarded for mining a variety of materials, including: diamond, gold, iron, emerald, coal, redstone. Higher valued ores will be more scarce within our grid.
```
Comment (lilphunky) [inline README.md:43]: Include lava -5 reward

File: README.md [hunk 3]
```diff
@@ -1,8 +1,73 @@
...
+| Gold | 5 | 6% |
+| Iron | 4 | 10% |
+| Emerald | 3 | 13% |
+| Coal | 2 | 16% |
+| Redstone | 0.1 | 20% 
+
+<br> 
+
+# Results
+After training our agent, there will be 6 graphs of different resources to show how well it mined each type of ores.
```
Comment (lilphunky) [inline README.md:58]: ore*

**Review by lilphunky — CHANGES_REQUESTED** [review]
Minor changes

File: README.md [hunk 4]
```diff
@@ -1,8 +1,75 @@
-![logo](docs/images/title.png)
+<p align="center">
+  <img src="docs/images/title.png">
+</p>
+
 ![portal](docs/images/background.jpg)
 
-Speed Miners will use Reinforcement Learning to gather as many resources in Malmo (Minecraft). The Minecraft character Steve will spawn with a diamond pickaxe to mine and gather as many coal, lapiz lazuli, redstone, emerald, iron, gold, and diamond within the map.  We will create an environment filled with Lava, Stone, Coal Ores, Lapiz Lazuli Ores, Redstone Ores, Emerald Ores, Iron Ores, Gold Ores, and Diamonds Ores. Steve will need to explore the map and mine different types of blocks until he is trained to mine efficiently. 
+Speed Miners will use Reinforcement Learning to gather as many resources in Malmo (Minecraft). The Minecraft character Steve will spawn with a diamond pickaxe to mine and gather as many coal, lapiz lazuli, redstone, emerald, iron, gold, and diamond within the map.  We will create an environment filled with Lava, Stone, Coal Ores, Redstone Ores, Emerald Ores, Iron Ores, Gold Ores, and Diamonds Ores. Steve will need to explore the map and mine different types of blocks until he is trained to mine efficiently.
```
Comment (erichsueh3) [inline README.md:7]: remove "lapiz lazuli"

File: README.md [hunk 5]
```diff
@@ -1,8 +1,75 @@
...
+
+```
+self.blocks_dict = {
+            "redstone_ore": 1,
+            "coal_ore": 3,
+            "emerald_ore": 4,
+            "iron_ore": 5,
+            "gold_ore": 6,
+            "diamond_ore": 7,
+            "lava": -1
```
Comment (erichsueh3) [inline README.md:31]: Add "flowing_lava": -1 to the dict

File: README.md [hunk 6]
```diff
@@ -1,8 +1,75 @@
...
+
+![ores](docs/images/ores.jpg)
+
+<br>
+
+# Rewards
+Our agent will be rewarded for mining a variety of materials, including: diamond, gold, iron, emerald, coal, redstone. Higher valued ores will be more scarce within our grid.
+
+
+| Materials | Rewards | Density |
```
Comment (erichsueh3) [inline README.md:47]: Should modify values for density after removal of lapis (refer to table in final report PR #33)

**Review by erichsueh3 — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -45,17 +47,18 @@ Our agent will be rewarded for mining a variety of materials, including: diamond
...
-| Gold | 5 | 6% |
+| Diamond | 6 | 2% |
+| Gold | 5 | 7% |
 | Iron | 4 | 10% |
 | Emerald | 3 | 13% |
-| Coal | 2 | 16% |
-| Redstone | 0.1 | 20% 
+| Coal | 2 | 17% |
+| Redstone | 0.1 | 25% |
+| Lava | -5 | 20%
```
Comment (lilphunky) [inline README.md:56]: Where did 20% lava come from?

**Pull request merged**